src/app/components/vertical-tabs/index.html

```diff
@@ -20,16 +20,44 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Vertical tab properties">
+    <sky-demo-page-property
+      propertyName="tabHeading"
+    >
+    Specifies the tab header.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
     >
-    Indicates whether the tab is active when the tabset loads.
+    Indicates whether the tab is active when the tabset loads. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="tabHeading"
+      propertyName="ariaControls"
+      isOptional="true"
     >
-    Specifies the tab header.
+    Specifies an ID to identify the element that contains the content the vertical tab exposes. This property supports <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure">accessibility rules for disclosures</a>. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="ariaInvalid"
+      isOptional="true"
+    >
+    Specifies which values are invalid in the tab's content when the tabset is part of a form. This property accepts <stache-code>boolean</stache-code> values.
+    <sky-demo-page-property
+      propertyName="ariaRole"
+      isOptional="true"
+    >
+    Specifies an ARIA role for the tab to support accessibility by indicating how the tab functions and what it controls. For information about ARIA roles, see the <a href="https://www.w3.org/WAI/PF/aria/roles">WAI-ARIA roles model</a>. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="ariaRequired"
+      isOptional="true"
+    >
+    Specifies which values are required for the form to be complete when the tabset is part of a form. This property accepts <stache-code>boolean</stache-code> values.
```

Comment:
To follow the style pattern with boolean properties, change "Specifies which" to "Indicates whether." I'd also add "the" before "values." ... And based on the note in https://github.com/blackbaud/skyux2/issues/1840, I think I'd add "in the tabset" after "values" and delete "for the form to be complete." I think it reads more naturally that way, and I think the angle about completing the form is implied by saying the fields are required.